packages/storybook/stories/ChatComposite/snippets/JoinExistingChatThread.tsx

```diff
@@ -11,19 +11,25 @@ export const JoinExistingChatThread: () => JSX.Element = () => {
   const knobs = useRef({
     endpointUrl: text('Azure Communication Services endpoint URL', '', 'External chat'),
     threadId: text('Existing thread', '', 'External chat'),
+    userId: text('User identifier for user', '', 'External chat'),
```

Comment:
nit: do we really need the 'for user' part here?

Reply:
Just trying to be consistent with the wording for the other knobs below.